app/config/sources.inc.php

```diff
@@ -806,6 +810,7 @@
         'ads/ios_android_apr2016.lang'         => ['de', 'es-ES', 'fr', 'pl'],
         'emails/2016/fundraising_email_1.lang' => ['de', 'es', 'fr', 'it'],
         'emails/2016/fundraising_email_2.lang' => ['de', 'es', 'fr', 'it'],
+        'emails/2016/fundraising_email_3.lang' => ['de', 'es', 'fr'],
```

Comment:
Yeah, we realized we have very few Italian people in our list, so it’s not really worth it to make you guys translate this email